src/app/lib/config/services/afaanoromoo.ts

```diff
@@ -159,7 +159,7 @@ export const service: DefaultServiceConfig = {
                   "Marsariitiin keenya akka hojjatu taasisuuf, fedhii keessan malee odeeffannoo waa'ee meeshaa itti fayyadamtanii muraasa ni olkeenya.",
                 para4: {
                   text: 'Marsariitiin keenyaa akka hojjatuuf odeeffannoo murteessoo meeshaa itti fayyadamtanii nuti olkeenyuu irratti caalaa dubbisaa.',
-                  url: 'https://www.bbc.co.uk/usingthebbc/strictly-necessary-cookies/',
+                  url: 'https://www.bbc.com/usingthebbc/strictly-necessary-cookies/',
```

Comment:
This URL returns a 404, so needs to be updated across the board 🤦 
https://www.bbc.com/usingthebbc/cookies/strictly-necessary-cookies/

Reply:
Thanks for catching this Karina.
I'll update the configs with the correct URL.